bin/filter.rb

```diff
@@ -56,6 +56,13 @@ def process(data, force_header: nil, format_for_frontpage: false)
   doc.css("strong:nth-child(1)").each do |node|
     id = node.content.downcase.gsub(/[^a-z0-9-]+/, '-').gsub(/^-|-$/, '')
     node["id"] = id if id.length > 1
+
+    if (format_for_frontpage == false)
+      anchor_link = "<a href=\"##{id}\" style=\"position: absolute; left: -20px; color: var(--menu-text); text-decoration: none;\">&para;</a>"
```

Comment:
(This is okay only because we know id cannot have a quote in it since we scrubbed it on line 57.)

Reply:
Oh right, we said there was a default for `format_for_frontpage` to `false`

Reply:
You lost the `node.parent[:style] = ` part.

Reply:
Dur.